apps/condo/domains/organization/schema/OrganizationEmployee.js

```diff
@@ -167,7 +167,7 @@ const OrganizationEmployee = new GQLListSchema('OrganizationEmployee', {
                             organization: { id: organizationId },
                         })
 
-                        if (!employeeRole) {
+                        if (!employeeRole || employeeRole.deletedAt) {
```

Comment:
Why no to add `deletedAt: null` into the search conditions above?

Reply:
I didn't think about it, the result will still be the same :)